submissions/iva-stasia/oop-exercise/index.js

```diff
@@ -0,0 +1,75 @@
+import { print } from './js/lib.js';
+
+class Inhabitant {
+   constructor(species, name, gender, legs, saying) {
+      this.species = species,
```

Comment:
Why commas?

Reply:
I forgot that commas are not used in classes and was confused with the syntax of objects. Fixed in my repo.